src/main/resources/api/service_openapi.yaml

```diff
@@ -175,6 +175,49 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{id}/google:
+    parameters:
+      - $ref: '#/components/parameters/Id'
+    post:
+      summary: Create a google context for the workspace.
+      operationId: createGoogleContext
+      tags: [Workspace]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateGoogleContextRequestBody'
+      responses:
+        '202':
+          description: Redirect for profile successful creation
+          headers:
+            location:
+              description: url for the job result
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JobModel'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
```

Comment:
Both operations can also throw a 404 if the workspace doesn't exist (or the user has no permission on it)

Reply:
Done.